cmd/modelcmd/controller.go

```diff
@@ -170,11 +171,24 @@ func (c *ControllerCommandBase) SetControllerName(controllerName string, allowDe
 func (c *ControllerCommandBase) ControllerName() (string, error) {
```

Comment:
initially, I wanted to pass `*cmd.Context` into this and the other function below.
Problem is that this leads to a whack a mole game. We have too many places which are either using it directly in the `run` or indirectly through other places. Injecting `context` everywhere is a lot of work.